### DISCUSSION TITLE
Using Adguard Home only on specific Websites

### DISCUSSION THREAD

**Discussion by NeXrom** [AdguardTeam/AdGuardHome#4947] [Q&A]
Hello there,

im using Adguard home over the Addon on my home Assistant Instance on a raspberry Pi.

Is it possible to enable Adguard only on certain websites? I can only find an option to exclude specific websites :(

**Comment by NeXrom** [comment]
so i kinda think than that is not possible

**Reply by andekande** [reply]
you won't need AdGuard for the default censored and ad bloated internet
